### PR TITLE
upd(firefox-deb): `105.0` -> `107.0.1`

### PR DESCRIPTION
**Pull request by soumyatheman**

### CODE REVIEW

File: packages/firefox-deb/firefox-deb.pacscript [hunk 1]
```diff
@@ -1,9 +1,10 @@
 name="firefox-deb"
 gives="firefox"
-version="105.0"
+version="107.0.1"
 replace="${gives}"
-url="http://launchpadlibrarian.net/624606759/firefox_${version}+build2-0ubuntu0.20.04.1_amd64.deb"
+#url="http://launchpadlibrarian.net/624606759/firefox_${version}+build2-0ubuntu0.20.04.1_amd64.deb"
```
Comment (Henryws) [inline packages/firefox-deb/firefox-deb.pacscript:5]: What's this comment for?

Comment (soumyatheman) [reply packages/firefox-deb/firefox-deb.pacscript:5]: In future if launchpadlibrarian is needed, the number, id number is necessary

**Review by Henryws — COMMENTED** [review]
@laniusone are you ok with changing maintainership?

**Comment by soumyatheman** [comment]
> @laniusone are you ok with changing maintainership?

obviously, that's nothing tough

**Comment by soumyatheman** [comment]
He is not updating the pkg for a long time, but many use that

**Comment by soumyatheman** [comment]
> > > @laniusone are you ok with changing maintainership?
> > 
> > 
> > obviously, that's nothing tough
> 
> I wasn't asking _you_, I was asking Laniusone.

Sorry, My bad

**Comment by soumyatheman** [comment]
Merge this please if everything is okay

**Review by Henryws — APPROVED** [review]

**Comment by laniusone** [comment]
Sorry, I've missed the notifications. It's fine of course.

**Pull request merged**